### ISSUE TITLE
[jshint] Bad option: 'overrides'

### ISSUE THREAD

**Issue by s-rodriguez** [Microsoft/vscode-jshint#52]
Since some recent update, this error keeps popping up: "**[jshint] Bad option: 'overrides'**". 

`overrides` comes as a [special option](http://jshint.com/docs/cli/#special-options) in the _cli_, same as `extends` that is indeed supported. 

I have the following section in my `.jshintrc`:

```
"overrides": {
  "test/**": {
    "mocha":   true,
    "globals": {
      "faker":  true,
      "should": false
    }
  },

  "test/support/factories/**": {
    "globals": {
      "_":         false,
      "Factory":   false,
      "faker":     false,
      "multiline": false,
      "util":      false
    }
  }
}
```

Has this been changed on purpose? Any idea on how can I fix it?

**Comment by RMacfarlane** [comment]
Support for `overrides` is missing, the option is being passed directly to the jshint api instead of being handled separately like `extends`. I'm not sure why this error would only start showing up now.

To fix it, the [server](https://github.com/Microsoft/vscode-jshint/blob/master/jshint-server/src/server.ts) needs to handle `overrides` when reading options.

**Comment by RMacfarlane** [comment]
@s-rodriguez Just pushed a change to add support for this, I'll publish a new version of the extension by the end of the week

**Comment by s-rodriguez** [comment]
@RMacfarlane excellent! Thanks a lot!

**Comment by RMacfarlane** [comment]
Published version 0.10.18!
